diff -urN aMule-AdunanzA-2010.1-2.2.6/src/amule.cpp aMule-AdunanzA-2010.1-2.2.6-patched/src/amule.cpp
--- aMule-AdunanzA-2010.1-2.2.6/src/amule.cpp	2010-02-04 03:14:39.000000000 +1000
+++ aMule-AdunanzA-2010.1-2.2.6-patched/src/amule.cpp	2011-10-15 01:00:41.283314653 +1100
@@ -556,7 +556,7 @@
 #endif
 
 	if ( !cmdline.Found(wxT("disable-fatal")) ) {
-#ifndef __WXMSW__
+#if wxUSE_ON_FATAL_EXCEPTION
 		// catch fatal exceptions
 		wxHandleFatalExceptions(true);
 #endif
